shared/team-building/container.tsx

```diff
@@ -355,10 +355,12 @@ const deriveRolePickerArrowKeyFns = memoize(
 const deriveOnChangeService = memoize(
   (
     onChangeService: OwnProps['onChangeService'],
+    incFocusInputCounter: OwnProps['incFocusInputCounter'],
     search: (text: string, service: Types.ServiceIdWithContact) => void,
     searchString: string
   ) => (service: Types.ServiceIdWithContact) => {
     onChangeService(service)
+    incFocusInputCounter()
```

Comment:
The ticket mentions not doing this on mobile, currently it does, intended?

Reply:
As I understand it, the keyboard shouldn't pop up on the keybase & contacts screen by default, but _should_ pop up when switching to other tabs. @keybase/design might have thoughts on that?